### PR TITLE
Remove PackageImports

### PR DESCRIPTION
**Pull request by natefaubion**
As mentioned in #3820, this is necessary to get `ghci` tooling to work in the multi-package setup.

### CODE REVIEW

File: lib/purescript-ast/src/Language/PureScript/Traversals.hs [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-maybeM _ Nothing = pure Nothing
-maybeM f (Just a) = Just <$> f a
-
-eitherM :: (Applicative f) => (a -> f c) -> (b -> f d) -> Either a b -> f (Either c d)
-eitherM f _ (Left a)  = Left  <$> f a
-eitherM _ g (Right b) = Right <$> g b
-
-defS :: (Monad m) => st -> val -> m (st, val)
-defS s val = return (s, val)
+-- | Common functions for implementing generic traversals
```
Comment (joneshf) [inline lib/purescript-ast/src/Language/PureScript/Traversals.hs]: I assume the reason all these lines changed is line-endings? If so, that seems fine.

Comment (natefaubion) [reply lib/purescript-ast/src/Language/PureScript/Traversals.hs]: Yes, I assume that's it.

**Comment by natefaubion** [comment]
Is a failure like this expected? Seems like one of the jobs isn't event starting.

**Comment by joneshf** [comment]
There was an [incident with macOS builds](https://www.traviscistatus.com/incidents/7mbxh26jk3sl). Seems to be resolved just recently. Maybe a restart now will get it going?

**Pull request merged**